packages/auth0-fastify/EXAMPLES.md

```
@@ -273,6 +278,243 @@ fastify.post('/custom-token-exchange', async (request, reply) => {
+```ts
+const STT_TOKEN_TYPE = 'urn:auth0:params:oauth:token-type:session_transfer_token';
+
+if (result.issuedTokenType !== STT_TOKEN_TYPE) {
+  // Not a session transfer token. Do not treat it as one.
+  throw new Error(`Unexpected issued token type: ${result.issuedTokenType}`);
+}
+```
+
+> [!IMPORTANT]
+> An **actor is mandatory** for an STT. That is what makes this auditable impersonation ("X acting as Y") rather than a silent account takeover. If you pass no explicit `actor` and no usable session ID token can be resolved (no logged-in agent, or an expired ID token with no refresh token), the SDK throws a `TokenExchangeError` with code `actor_unavailable` **before any network call**. When the agent's session ID token has expired and a refresh token is available, the SDK refreshes it automatically and persists the refreshed session.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)